content/asa/0007-fluxos-maximos.md

```diff
@@ -597,11 +597,11 @@ RelabelToFront(G, s, t)
     u = u.next // next -> proximo vertice de L
 ```
 
-A aplicação do algoritmo é trivial, porém é bastante fácil enganarmo-nos num passo intermédio (costumam ser uns quantos, e é fácil haver detalhes a passar despercebidos). Abaixo encontramos um exemplo do decorrer do mesmo (retirado dos slides):
+A aplicação do algoritmo é trivial, porém é bastante fácil enganámo-nos num passo intermédio (costumam ser uns quantos, e é fácil haver detalhes a passar despercebidos). Abaixo encontramos um exemplo do decorrer do mesmo (retirado dos slides):
```

Comment:
Is it correct?

Reply:
I think :thinking:

Isso ou como estava

Reply:
Como estava estava bem yah
```suggestion
A aplicação do algoritmo é trivial, porém é bastante fácil enganarmo-nos num passo intermédio (costumam ser uns quantos, e é fácil haver detalhes a passar despercebidos). Abaixo encontramos um exemplo do decorrer do mesmo (retirado dos slides):
```